DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Raw/LocationConversionToSpeckle.cs

```diff
@@ -1,31 +1,33 @@
-using Autodesk.Revit.DB;
+using Objects;
 using Speckle.Converters.Common.Objects;
 using Speckle.Converters.Common;
 using Speckle.Core.Models;
 
 namespace Speckle.Converters.RevitShared.ToSpeckle;
 
-[NameAndRankValue(nameof(DB.Location), 0)]
-public class LocationConversionToSpeckle : BaseConversionToSpeckle<DB.Location, Base>
+public class LocationConversionToSpeckle : IRawConversion<DB.Location, Base>
 {
-  private readonly IRawConversion<DB.Curve> _curveConverter;
+  private readonly IRawConversion<DB.Curve, ICurve> _curveConverter;
   private readonly IRawConversion<DB.XYZ, SOG.Point> _xyzConverter;
 
   // POC: review IRawConversion<TIn> which always returns a Base, this is ToSpeckle, so... this breaks
   // the meaning of IRawConversion, it could be IToSpeckleRawConversion
   // also a factory type
-  public LocationConversionToSpeckle(IRawConversion<Curve> curveConverter, IRawConversion<XYZ, SOG.Point> xyzConverter)
+  public LocationConversionToSpeckle(
+    IRawConversion<DB.Curve, ICurve> curveConverter,
+    IRawConversion<DB.XYZ, SOG.Point> xyzConverter
+  )
   {
     _curveConverter = curveConverter;
     _xyzConverter = xyzConverter;
   }
 
-  public override Base RawConvert(DB.Location target)
+  public Base RawConvert(DB.Location target)
   {
     return target switch
     {
-      LocationCurve curve => _curveConverter.ConvertToBase(curve.Curve),
-      LocationPoint point => _xyzConverter.RawConvert(point.Point),
+      DB.LocationCurve curve => (_curveConverter.RawConvert(curve.Curve) as Base)!, // POC: ICurve and Base are not related but we know they must be, had to soft cast and then !.
```

Comment:
Okay me reaction is a bit :-O

Reply:
This is horrible but it is coming from `Objects` and I didn't want to touch it, nor am I certain how we should fix it. Let's have a chat at some point regarding this.

We've been thinking about having `IBase` or `ISpeckleObject` to be the backing interface for `Base`, which would enable this but with some pretty big breaking changes.